completers/nix_completer/cmd/build.go

```diff
@@ -33,8 +33,11 @@ func init() {
 	rootCmd.AddCommand(buildCmd)
 
 	carapace.Gen(buildCmd).FlagCompletion(carapace.ActionMap{
-		"out-link": carapace.ActionFiles(),
-		"profile":  carapace.ActionFiles(),
+		"inputs-from":         nix.ActionFlakes(),
+		"out-link":            carapace.ActionFiles(),
+		"output-lock-file":    carapace.ActionFiles(),
+		"profile":             carapace.ActionFiles(),
+		"reference-lock-file": carapace.ActionFiles("nix"),
```

Comment:
Lock files always end with `nix`?

Reply:
No, I was thinking of `flake.nix` files, i've updated the PR for the lock file suffix to be `lock` instead, since lock files should be generated via nix commands I feel it's a safe choice to assume they have the .lock extension